spec/requests/relationships_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Relationships', type: :request do
+  describe 'follow user' do
+    image_path = File.join(File.dirname(__FILE__), '..', 'images', 'download.jpeg')
+    let(:follower) { create :user }
+    let(:followed) { create :user }
+
+    let(:relationship) { Relationship.create(follower:, followed:) }
+
+    before do
+      follower.avatar.attach(io: File.open(image_path), filename: 'download.jpeg', content_type: 'image/jpeg')
+      followed.avatar.attach(io: File.open(image_path), filename: 'download.jpeg', content_type: 'image/jpeg')
+      sign_in follower
+    end
+
+    it 'verifies response is success' do
+      post relationships_path, params: { relationship: { followed_id: followed.id } }
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'verifies relationship count increases' do
+      count = Relationship.count
+      post relationships_path, params: { relationship: { followed_id: followed.id } }
+      expect(Relationship.count).to be(count + 1)
+    end
+
+    it 'verifies follower follows the followed' do
+      post relationships_path, params: { relationship: { followed_id: followed.id } }
+      expect(follower.following?(followed)).to be_truthy
+    end
+  end
+
+  describe 'unfollow user' do
+    image_path = File.join(File.dirname(__FILE__), '..', 'images', 'download.jpeg')
+    let(:follower) { create :user }
+    let(:followed) { create :user }
+
+    let(:relationship) { Relationship.create(follower:, followed:) }
+
+    before do
+      follower.avatar.attach(io: File.open(image_path), filename: 'download.jpeg', content_type: 'image/jpeg')
+      followed.avatar.attach(io: File.open(image_path), filename: 'download.jpeg', content_type: 'image/jpeg')
+      sign_in follower
+      post relationships_path, params: { relationship: { followed_id: followed.id } }
```

Comment:
```suggestion
```
This line will create the same relationship as the one you created in the 'let' statement.

Reply:
You just need to initialize it by calling the variable